decidim-proposals/app/jobs/decidim/proposals/notify_proposals_mentioned_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    class NotifyProposalsMentionedJob < ApplicationJob
+      def perform(comment, proposal_metadata)
```

Comment:
Maybe instead of the instances we should pass the ids of the comment and proposals